src/main/java/org/wso2/carbon/connector/iso8583/ISO8583PackagerFactory.java

```diff
@@ -16,10 +16,13 @@
 
 package org.wso2.carbon.connector.iso8583;
 
+import org.apache.synapse.SynapseException;
+import org.jpos.iso.ISOBasePackager;
 import org.jpos.iso.ISOException;
 import org.jpos.iso.ISOPackager;
 import org.jpos.iso.packager.GenericPackager;
 
+
```

Comment:
This line is unnecessary. Can we remove it?

Reply:
removed